eegnb/experiments/Experiment.py

```diff
@@ -0,0 +1,133 @@
+""" 
+Initial run of the Experiment Class Refactor base class
+
+Specific experiments are implemented as sub classes that overload a load_stimulus and present_stimulus method
+
+
+Running each experiment:
+obj = VisualP300({parametrs})
+obj.present()
+"""
+
+from abc import ABC, abstractmethod
+from psychopy import prefs
+#change the pref libraty to PTB and set the latency mode to high precision
+prefs.hardware['audioLib'] = 'PTB'
+prefs.hardware['audioLatencyMode'] = 3
+
+import os
+from time import time
+from glob import glob
+from random import choice
+from optparse import OptionParser
+import random
+
+import numpy as np
+from pandas import DataFrame
+from psychopy import visual, core, event
+
+from eegnb import generate_save_fn
+from eegnb.devices.eeg import EEG
+
+class Experiment:
```

Comment:
see comment below. I think 

```python
class ExperimentBase
```

or 

```python
class BaseExperiment
```

Reply:
If the other experiments are named `P300Experiment` for example, then it should prob be `BaseExperiment`